ts/components/RemindUpdatePagoPaVersionOverlay.tsx

```diff
@@ -0,0 +1,117 @@
+import I18n from "i18n-js";
+import { Millisecond } from "italia-ts-commons/lib/units";
+import { Button, Content, H2, Text, View } from "native-base";
+import * as React from "react";
+import { Image, Linking, Platform, StyleSheet } from "react-native";
+import { connect } from "react-redux";
+import customVariables from "../theme/variables";
+
+type State = { hasError: boolean };
+
+const styles = StyleSheet.create({
+  container: {
+    alignSelf: "center",
+    marginTop: customVariables.contentPadding,
+    padding: customVariables.contentPadding
+  },
+  imageChecked: {
+    alignSelf: "center"
+  },
+  emailTitle: {
```

Comment:
It should have a more appropriate name, since email is not interested in this flow